Gems/ROS2/Code/Source/RobotImporter/URDF/UrdfParser.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/std/string/string.h>
+#include <console_bridge/console.h>
+#include <sstream>
+#include <urdf_parser/urdf_parser.h>
+
+namespace ROS2
+{
+    //! Class for parsing URDF data.
+    class UrdfParser
```

Comment:
Similar to previous suggestions, this should be a namespace instead of a class if its just used as a holder for a group of methods.